### ISSUE TITLE
Sound test button

### ISSUE THREAD

**Issue by yuriykulikov** [yuriykulikov/AlarmClock#378]
# Sound test button

## Summary

Sometimes the user has his phone in silent mode or the volume is set to 0. In this case it would be nice to display a warning and a test button to test the sound.

## Motivation
Nothing is worse than a silent alarm.

## Description
Show a snackbar or a warning in the main screen

**Comment by kolanse** [comment]
Please i do not fully get . is it when the app is opened .. if the app detects that the alarm sound is on silent then the warning should be displayed ?.

**Comment by adityabanotra** [comment]
Great, I'll work on it.

**Comment by yuriykulikov** [comment]
> Please i do not fully get . is it when the app is opened .. if the app detects that the alarm sound is on silent then the warning should be displayed ?. 
> 

Yes, something along those lines. I do not have a clear picture yet, open for discussion 😀

**Comment by adityabanotra** [comment]
![WhatsApp Image 2021-08-25 at 00 05 53](https://user-images.githubusercontent.com/71438721/130671504-2b209fc4-88be-4299-980f-efdb4b77f7e3.jpeg)

Does this look good? I am working on it. I have some other improvements in mind that we can take incrementally .

**Comment by kolanse** [comment]
@adityabanotra  looks Good to me. what other improvements did you have in mind? . The snackbar only shows in this fragment right ?

**Comment by kolanse** [comment]
> > Please i do not fully get . is it when the app is opened .. if the app detects that the alarm sound is on silent then the warning should be displayed ?.
> 
> Yes, something along those lines. I do not have a clear picture yet, open for discussion 😀

Yeah i guess it could work .

**Comment by yuriykulikov** [comment]
Looks good. How about another color (orange or yellow) and an additional button "ignore"?
